templates/kpi/settings.py.j2

```diff
@@ -765,6 +765,5 @@ if MONGO_DATABASE.get('USER') and MONGO_DATABASE.get('PASSWORD'):
 else:
     MONGO_CONNECTION_URL = "mongodb://%(HOST)s:%(PORT)s" % MONGO_DATABASE
 
-MONGO_CONNECTION = MongoClient(
-    MONGO_CONNECTION_URL, j=True, tz_aware=True, connect=False)
-MONGO_DB = MONGO_CONNECTION[MONGO_DATABASE['NAME']]
+MONGO_CONNECTION = None
+MONGO_DB = None
```

Comment:
Could we make the two values configurable via the vars with the default being None... Would make it easier in the future to utilize this functionality if the need arises

```
...
{% if enable_mongo_connection %}
MONGO_CONNECTION = MongoClient({{ mongo_connection_ url }}, .....
{% endif %}
...
```